### DISCUSSION TITLE
math: can't produce $ sign in math mode

### DISCUSSION THREAD

**Discussion by nschloe** [community/community#17116] [Other Feature Feedback, Questions, & Ideas]
It works in display math
```
$$
x = \$
$$
```

$$
x = \$,
$$

but not in inline math
```
$x = \$$
```
$x = \$$.

**Comment by Simran-B** [comment]
It seems to work if you escape the dollar sign with two backslashes and add spaces around the formula:

```
$ x = \\$ $
```

$ x = \\$ $

but that's not how it should work.

The following two single-line variants for display math work, too:

```
$$x = \\$$$

$$ x = $ $$
```

$$x = \\$$$

$$ x = $ $$

The second one seems to work similarly to inline code in Markdown: ``` ``  backtick ` `` ```

**Reply by nschloe** [reply]
Ah, right, putting spaces around the doubly-escaped `$` works.

> but that's not how it should work.

True.

**Comment by nschloe** [comment]
Does does work now, but the spacing is wrong.

**Comment by nschloe** [comment]
Fixed by using the new backtick-math delimiters,

> $`x = \$`$
